### PR TITLE
Add the folder for the training at ood-2022

### PR DESCRIPTION
**Pull request by atoato88**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
This PR adds the folder for k8s upstream training held at ood-2022(Okinawa Open Days 2022).

**Which issue(s) this PR fixes**:
NONE

**Special notes for your reviewer**:
Please check your "Focus on, Role." sectoin in the `README.md`, and update that section if there are additional roles or update old roles.

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:
NONE

### CODE REVIEW

**Comment by atoato88** [comment]
/cc @t-inu 
/cc @s-kawamura-w664 
/cc @YuikoTakada 
/cc @ziyi-xie 
/cc @oomichi 
/cc @shu-mutou 

Please check your `Focus on, Role.` on `講師（Facilitators）` section in the [README.md](https://github.com/kubernetes-sigs/contributor-playground/blob/494aeeb5177ca3bd9b8bd9b77301207f99f01d56/japan/ood-2022/README.md), and suggest the update if there are additional roles or old roles.

File: japan/ood-2022/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+| ------------- | ------------- | ------------- | ------------- | ------------- |
+|<a href="https://github.com/oomichi"><img src="https://avatars.githubusercontent.com/u/1287154?s=50"></a>| <a href="https://github.com/oomichi">Kenichi Omichi</a> | @oomichi | NEC | - sig-testing<br> - Approver of k8s.io/kubernetes/test<br> - Owner of kubernetes-sigs/kubespray |
+|<a href="https://github.com/atoato88"><img src="https://avatars.githubusercontent.com/u/748740?s=50"></a>| <a href="https://github.com/atoato88">Akihito Inoh</a> | @akihito-inou(atoato88) | NEC Solution Innovators | - sig-cluster-lifecycle<br> - Approver of kubernetes-sigs/kubebuilder-declarative-pattern <br> - k/dashboard i18n ja |
+|<a href="https://github.com/shu-mutou"><img src="https://avatars.githubusercontent.com/u/12838129?s=50"></a>| <a href="https://github.com/shu-mutou">Shu Muto</a> | @Shu Muto | NEC Solution Innovators | - Chair of sig-ui<br> - Approver of k/dashboard |
+|<a href="https://github.com/YuikoTakada"><img src="https://avatars.githubusercontent.com/u/6335296?s=50"></a>| <a href="https://github.com/YuikoTakada">Yuiko Mouri</a> | @YuikoTakada | NEC Solution Innovators | - sig-node |
```
Comment (YuikoTakada) [inline japan/ood-2022/README.md:25]: Could you please add sig-storage also?

**Review by YuikoTakada — COMMENTED** [review]
Thank you for the PR. I added just one comments. PTAL

File: japan/ood-2022/README.md (hunk 1 above)
Comment (atoato88) [reply japan/ood-2022/README.md:25]: I'll do that.

Comment (atoato88) [reply japan/ood-2022/README.md]: Done.
Please check the update.

**Comment by oomichi** [comment]
Thank you for organizing.

LGTM

**Comment by YuikoTakada** [comment]
Thank you for updating. LGTM!

**Comment by shu-mutou** [comment]
LGTM, Thanks!!

**Comment by oomichi** [comment]
@t-inu @s-kawamura-w664 @ziyi-xie
Please check your descriptions and put `LGTM` if your own is fine.

File: japan/ood-2022/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+|<a href="https://github.com/shu-mutou"><img src="https://avatars.githubusercontent.com/u/12838129?s=50"></a>| <a href="https://github.com/shu-mutou">Shu Muto</a> | @Shu Muto | NEC Solution Innovators | - Chair of sig-ui<br> - Approver of k/dashboard |
+|<a href="https://github.com/YuikoTakada"><img src="https://avatars.githubusercontent.com/u/6335296?s=50"></a>| <a href="https://github.com/YuikoTakada">Yuiko Mouri</a> | @YuikoTakada | NEC Solution Innovators | - sig-storage<br> - sig-node |
+|<a href="https://github.com/ziyi-xie"><img src="https://avatars.githubusercontent.com/u/92832323?s=50"></a>| <a href="https://github.com/ziyi-xie">Xie Ziyi</a> | @XIE ZIYI | NEC Solution Innovators | - sig-docs<br> - k/dashboard i18n ja  |
+|<a href="https://github.com/s-kawamura-w664"><img src="https://avatars.githubusercontent.com/u/76156228?s=50"></a>| <a href="https://github.com/s-kawamura-w664">Shinji Kawamura</a> | @Shinji Kawamura (s-kawamura-w664)  | NEC Solution Innovators | - sig-node |
```
Comment (s-kawamura-w664) [inline japan/ood-2022/README.md:27]:
```suggestion
|<a href="https://github.com/s-kawamura-w664"><img src="https://avatars.githubusercontent.com/u/76156228?s=50"></a>| <a href="https://github.com/s-kawamura-w664">Shinji Kawamura</a> | @Shinji Kawamura (s-kawamura-w664)  | NEC Solution Innovators | - sig-docs<br> - sig-node |
```
Could you please add sig-docs in my 'Focus on'.

**Review by s-kawamura-w664 — COMMENTED** [review]
Please confirm my one request.

**Comment by ziyi-xie** [comment]
LGTM, Thank you for the PR!

**Comment by t-inu** [comment]
LGTM, Thank you!

File: japan/ood-2022/README.md (hunk 2 above)
Comment (atoato88) [reply japan/ood-2022/README.md]: I've merged your suggestion, thank you.

**Comment by atoato88** [comment]
Now, I've gotten all of members review, and merged all suggestions.

@s-kawamura-w664 
Please lgtm label to this PR.

**Comment by s-kawamura-w664** [comment]
Thank you for updating.
I have confirmed that all members have completed to review.

/lgtm

**Pull request merged**